sdk/src/main/java/com/ifountain/opsgenie/client/model/BaseRequest.java

```diff
@@ -72,6 +76,94 @@ public Map<String,String> getReqHeadersForGetOrDelete(){
         return addGenieKey();
     }
 
+    public boolean isValidDate(String dateStr) {
+        try {
+            Instant instantDate = Instant.parse(dateStr);
+        }
+        catch (DateTimeParseException e){
+            return false;
```

Comment:
Can we please also log something may be an exception along with false value

Reply:
This method is not used anywhere. I'll remove it.